week7-18-03-12.md

```diff
@@ -0,0 +1,65 @@
+# Week 7
+
+## New Features
+
+### Adds Ability To Execute Shell Commands
+#### Will include or not include based on Tanner's response
+
+[vapor/core#100](https://github.com/vapor/core/pull/100)
```

Comment:
I think this is just an extension on the `Process` [in Foundation](https://developer.apple.com/documentation/foundation/process)

Reply:
Tanner confirmed that this is new:

https://github.com/vapor/core/pull/100#issuecomment-375071470

Reply:
Sweet! 👍